lib/sms_factor/webhooks.ex

```diff
@@ -0,0 +1,32 @@
+defmodule SMSFactor.Webhooks do
+  @moduledoc """
+  Wrappers around **Webhooks** section of SMSFactor API.
+  """
+
+  @typedoc """
+  Webhook Type can be any of the following :
+
+  - "MO"
+  - "DLR"
+  - "STOP"
+  - "CLICKER"
+  - "BALANCE_ALERT"
+  """
+  @type webhook_type() :: String.t()
+
+  @type webhook_params() :: %{webhook: %{type: webhook_type(), url: String.t()}}
+
+  @spec create_webhook(Tesla.Client.t(), webhook_params()) :: Tesla.Env.result()
+  def create_webhook(client, params), do: Tesla.post(client, "/webhook", params)
+
+  @spec retrieve_webhooks(Tesla.Client.t()) :: Tesla.Env.result()
+  def retrieve_webhooks(client), do: Tesla.get(client, "/webhook")
```

Comment:
Why "retrieve" instead of "get"?

Reply:
I wanted to be as close as possible to the API, including their wording.